updatem/orchestration/reboot.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+package orchestration
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/eclipse-kanto/update-manager/logger"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	procSysRqFile        = "/wproc/sys/kernel/sysrq"
```

Comment:
Please make this location customizable with environment variable or something like that.

Reply:
Added configuration using ENVs FILE_SYS_RQ, FILE_SYS_RQ_TRIGGER.
Also possible to use syscall.Reboot instead of Linux Magic System Request Key combination.